backend/src/browser.py

```diff
@@ -198,6 +192,28 @@ async def _install(self, artifact: str, name: str, version: str, hash: str):
                 if res.status != 200:
                     logger.error(f"Server did not accept install count increment request. code: {res.status}")
 
+        if res_zip and version == "dev":
+            with ZipFile(res_zip) as plugin_zip:
+                plugin_json_list = [file for file in plugin_zip.namelist() if file.endswith("/plugin.json")]
```

Comment:
the `plugin.json` should always be one sub folder deep within the zip. The other ones have no incidence in how a plugin is loaded.

```python
plugin_json_list = [file for file in plugin_zip.namelist() if file.endswith("/plugin.json") and file.count("/") == 1]
```